integration_tests/src/main/python/window_function_test.py

```diff
@@ -213,3 +213,54 @@ def test_window_aggs_for_ranges_of_dates(data_gen):
         '       range between 1 preceding and 1 following) as sum_c_asc '
         'from window_agg_table'
     )
+
+
+'''
+  Spark will drop nulls when collecting, but seems GPU does not, so exceptions come up.
+E      Caused by: java.lang.AssertionError:  value at 350 is null
+E          	at ai.rapids.cudf.HostColumnVectorCore.assertsForGet(HostColumnVectorCore.java:228)
+E         	at ai.rapids.cudf.HostColumnVectorCore.getInt(HostColumnVectorCore.java:254)
+E         	at com.nvidia.spark.rapids.RapidsHostColumnVectorCore.getInt(RapidsHostColumnVectorCore.java:109)
+E         	at org.apache.spark.sql.vectorized.ColumnarArray.getInt(ColumnarArray.java:128)
+
+  Now set nullable to false to pass the tests, once native supports dropping nulls, will set it to true.
+'''
+collect_data_gen = [
```

Comment:
Can we please add the a link to the follow on issue to fix collect_list so we don't collect nulls? That way we can update the tests to allow for nulls again?

Reply:
Sure.
The issue is here https://github.com/NVIDIA/spark-rapids/issues/1638 . 
Added two tests for nullabe and non-nullable columns separately, and skipped the former one with the tracking issue above.